sauce/features/budget/check-credit-balances/index.js

```diff
@@ -203,12 +201,14 @@ export class CheckCreditBalances extends Feature {
     return false;
   }
 
-  updateCreditBalances(name, difference) {
+  updateCreditBalances() {
     // eslint-disable-next-line no-alert
     if (ynabToolKit.options.QuickBudgetWarning && (!confirm('Are you sure you want to do this?'))) {
       return;
     }
 
+    let name = $(this).data('name');
+    let difference = $(this).data('difference');
```

Comment:
On these two lines, the $(this) keyword is the button again, which has data attached to it.